src/messages/variant.rs

```diff
@@ -25,21 +25,19 @@ use std::{
 /// Message variant
 pub enum Variant {
     /// Inform neighbours about our new section.
-    NeighbourInfo(EldersInfo),
+    NeighbourInfo {
+        /// `EldersInfo` of the neighbour section.
+        elders_info: EldersInfo,
+        /// Nonce that is derived from the incoming message that triggered sending this
+        /// `NeighbourInfo`. It's purpose is to make sure that `NeighbourInfo`s that are otherwise
```

Comment:
The idea was to make the typename highlighted (using backticks) in the rendered docs but also put it into plural which works better grammatically. I didn't want to backtick the "s" too as it's not part of the type name.

Reply:
Ah, I am ok with the backtick. I am just wondering about the `that are otherwise ...` part  :)

Reply:
I see. The full sentence is  "make sure that NeighbourInfo`s that are otherwise identical but triggered by different messages are not filtered out". Not sure what's wrong with it?

Reply:
you mean `ensure that identical NeighbourInfos but triggered by different messages are not filtered out` ?
the word of `otherwise` made me bit confused.  :(

Reply:
Yeah. Will change the wording.